src/improvements/template/TransferOwners.sol

```diff
@@ -0,0 +1,166 @@
+    address public permissionlessWETH;
+
+    /// @notice Stores the chain information after setup.
+    SuperchainAddressRegistry.ChainInfo internal _activeChainInfo;
+
+    /// @notice Returns the safe address string identifier.
+    function safeAddressString() public pure override returns (string memory) {
+        return "ProxyAdminOwner";
+    }
+
+    /// @notice Returns the storage write permissions required for this task.
+    function _taskStorageWrites() internal pure virtual override returns (string[] memory) {
+        string[] memory storageWrites = new string[](4);
+        storageWrites[0] = "DisputeGameFactoryProxy";
+        storageWrites[1] = "ProxyAdmin";
+        storageWrites[2] = "PermissionedWETH";
+        storageWrites[3] = "PermissionlessWETH";
+        return storageWrites;
+    }
+
+    /// @notice Sets up the template with the new owner from a TOML file.
+    function _templateSetup(string memory taskConfigFilePath) internal override {
+        super._templateSetup(taskConfigFilePath);
+        string memory toml = vm.readFile(taskConfigFilePath);
+        newOwner = abi.decode(vm.parseToml(toml, ".newOwner"), (address));
+
+        // Only allow one chain to be modified at a time with this template.
+        SuperchainAddressRegistry.ChainInfo[] memory _parsedChains =
+            abi.decode(vm.parseToml(toml, ".l2chains"), (SuperchainAddressRegistry.ChainInfo[]));
+        require(_parsedChains.length == 1, "Must specify exactly one chain id to transfer ownership for");
+        _activeChainInfo = _parsedChains[0]; // Store the ChainInfo struct
+
+        STORAGE_SETTER = abi.decode(vm.parseToml(toml, ".addresses.StorageSetter"), (address));
+        require(address(STORAGE_SETTER).code.length > 0, "Incorrect StorageSetter - no code at address");
+        require(StorageSetter(STORAGE_SETTER).version().eq("1.1.0"), "Incorrect StorageSetter version");
+    }
+
+    /// @notice Builds the actions for transferring ownership of the DisputeGameFactory, DWETH contracts and ProxyAdmin.
+    function _build() internal override {
+        ProxyAdmin proxyAdmin = ProxyAdmin(superchainAddrRegistry.getAddress("ProxyAdmin", _activeChainInfo.chainId));
+        address dgfProxy = superchainAddrRegistry.getAddress("DisputeGameFactoryProxy", _activeChainInfo.chainId);
+        permissionedWETH = _getDWETH("PermissionedWETH", _activeChainInfo.chainId);
+        permissionlessWETH = _getDWETH("PermissionlessWETH", _activeChainInfo.chainId);
+
+        // Get the current implementation of the DisputeGameFactory before the storage setter is set.
+        address originalDGFImpl = getEIP1967Impl(dgfProxy);
+        // Set initialized slot to zero.
+        setInitializedToZero(proxyAdmin, dgfProxy);
+        // Reinitialize the DisputeGameFactory with the new owner.
+        proxyAdmin.upgradeAndCall(
+            payable(dgfProxy), originalDGFImpl, abi.encodeCall(DisputeGameFactory.initialize, (newOwner))
+        );
```

Comment:
DGF is ownable so we can just do a `transferOwnership` call and remove the need for the storage setter

Reply:
done https://github.com/ethereum-optimism/superchain-ops/pull/972/commits/8e6a1f5c1a32e401946e174e0002e3e02f2ed76b